lang/en.lyaml

```diff
@@ -1517,6 +1517,10 @@ en:
         inactiveSecondary: "Run {{ command }} to remove inactive accounts from your CLI config"
         unableToDetermine: "Unable to determine if the portal is active"
         pak:
+          incomplete:
+            one: "Personal access key is valid. {{ count }} scope available, but not included in your key."
+            other: "Personal access key is valid. {{ count }} scopes available, but not included in your key."
+          incompleteSecondary: "To add the available scopes, run `hs auth` and re-authenticate your account with a new key that has those scopes. {{ link }}"
```

Comment:
We have a helper function called `uiCommandReference` that will format `hs auth` correctly.

Reply:
I'd also add some messaging here that the personal access key UI will have a full list of missing scopes, if we're not going to list them all out in the CLI.

Reply:
Not a bad idea, I added some extra copy. lmk what you think